allennlp/data/tokenizers/word_splitter.py

```diff
@@ -111,8 +115,13 @@ class JustSpacesWordSplitter(WordSplitter):
     tokens does not matter.  This will never result in spaces being included as tokens.
     """
     @overrides
-    def split_words(self, sentence: str) -> Tuple[List[str], List[Tuple[int, int]]]:
-        return sentence.split(), None
+    def split_words(self, sentence: str) -> List[Token]:
+        return list(map(Token, sentence.split()))
```

Comment:
another place to consider a list comprehension

Reply:
Done.